app/jobs/send_daily_triage_email_job.rb

```diff
@@ -28,4 +28,14 @@ def skip_daily_email?(user)
   def email_decider(user)
     @email_decider ||= EmailDecider.new(user.days_since_last_clicked, minimum_frequency: user.email_frequency)
   end
+
+  DEFAULT_EMAIL_TIME_OF_DAY = Time.utc(2000, 1, 1, 17, 00, 0)
+
+  def before_email_time_of_day?(user)
```

Comment:
The time column stored the time as well as the data 2000-1-1. 
```
irb(main):002:0> User.first.email_time_of_day.utc
 => 2000-01-01 17:00:00 UTC
```
This is comparing the time it is now for today, against the time that is set in the database (ignoring the data part).

If you know of a method to handle this, I'm open for suggestions.

It's why you also see this further down
```
Time.utc(now.year, now.month, now.day, time.hour, time.min, time.sec)
```